scripts/set-environment

```diff
@@ -66,6 +66,11 @@ fi
   echo 'export BEHAT_ADMIN_PASSWORD=$(openssl rand -base64 24)'
   echo 'export BEHAT_ADMIN_USERNAME=pantheon-ci-testing-$CI_BUILD_NUMBER'
   echo 'export BEHAT_ADMIN_EMAIL=no-reply+ci-$CI_BUILD_NUMBER@getpantheon.com'
+  echo "export MULTIDEV_SITE_URL='https://$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io/'"
+  echo "export LIVE_SITE_URL='https://live-$TERMINUS_SITE.pantheonsite.io/'"
+  echo "export ARTIFACTS_DIR='artifacts'" >> $BASH_ENV
+  echo "export ARTIFACTS_FULL_DIR='/tmp/artifacts'" >> $BASH_ENV
+  echo "export ARTIFACTS_DIR_URL='${CIRCLE_BUILD_URL}/artifacts/${CIRCLE_NODE_INDEX}/artifacts'" >> $BASH_ENV
```

Comment:
Circle-specific environment variables should not be used here. Define generic environment variable names and set them to their circle-provided value if not set.

Example:
```
CI_BUILD_URL=${CI_BUILD_URL:-$CIRCLE_BUILD_URL}
```

Reply:
That works for CircleCI but how would we define something generic on other providers?

Maybe we could check for a provider-specific variable, such as `CIRCLECI`, and set the defaults for each provider in their own block.

Reply:
I define the default variable names before calling this script. See https://github.com/pantheon-systems/example-drops-8-composer/pull/198/files#diff-81f62e874f3f2d77c10287105a377dceR12

Reply:
The script `.bitbucket/set-up-globals.sh` will then go away.